crates/sui-graphql-rpc/src/consistency.rs

```diff
@@ -98,7 +98,7 @@ pub(crate) fn build_objects_query(
     view: View,
     lhs: i64,
     rhs: i64,
-    page: &Page<Cursor>,
+    page: Option<&Page<Cursor>>,
```

Comment:
Immediately nervous about the implications of this change -- but will wait to see how it's used later down the line.

Reply:
So this is used to fetch domain expirations down the line, without applying pagination. 

I don't think that's a valid reason to make the limits imposed by pagination optional: Either we can apply a static upperbound to the query, in which case we can construct a `Page<Cursor>` to represent that, or we can't in which case we'll need to rethink the approach to make sure someone can't DoS the GraphQL service by constructing a SuiNS domain with some unboundedly many domain expirations that need fetching.

Reply:
@amnn This is actually used to do a "multi_get_objects" without the overhead of constructing a page (in our case, it fetches the name record, and the parent name record if the name is a subdomain). So the upper bound is 2 objects per query (It'd never be more than 2).

I'll defer to @wlmyng as we had some discussions on it and he'll elaborate better than me 😄

Reply:
So that means we're in the case where we can set a static upperbound, in which case we should be able trivially construct a `Page` that applies a limit of two, and we're all good!

Reply:
Makes sense to me - I've made the modification